.eslintrc.yaml

```diff
@@ -106,7 +106,7 @@ rules:
 
   # React Native.  This plugin isn't included in the airbnb config, and
   # nothing is enabled by default, so we enable its rules.
-  react-native/no-inline-styles: error
+  react-native/no-inline-styles: off # Noisy and a bad guide to rerendering optimizations.
   # react-native/no-unused-styles: error  # This is buggy on the `this.styles` pattern.
   # react-native/no-color-literals: error  # TODO eliminate these and enable.
```

Comment:
nit: the rule is already off by default, so can just delete the line; or make it like the two below it, with `error` but commented out.

Reply:
The next thing I notice here is that this means we're no longer using this ESLint plugin at all. :-) So should probably drop it entirely.

Looking back through the [list of all its rules](https://github.com/Intellicode/eslint-plugin-react-native/tree/master/docs/rules)... I think the only two we actually want are the two commented out below this one. Maybe we'd want to keep these commented-out config lines, as a reminder for a possible future where we try to bring one or both of those rules into use. But in the meantime, it'll be good to make `node_modules` a bit smaller, and `tools/test lint` perhaps a bit faster.